test/node/timestamp.test.ts

```diff
@@ -9,6 +9,27 @@ describe('Timestamp', () => {
     });
   });
 
+  describe('get i() and get t()', () => {
+    it('i returns lower bits', () => {
+      const l = new BSON.Long(1, 2);
+      const ts = new BSON.Timestamp(l);
+      expect(ts.i).to.equal(l.low);
+    });
+
+    it('t returns higher bits', () => {
+      const l = new BSON.Long(1, 2);
+      const ts = new BSON.Timestamp(l);
+      expect(ts.t).to.equal(l.high);
+    });
+
+    it('t and i always return unsigned values', () => {
+      const l = new BSON.Long(-1, -2);
+      const ts = new BSON.Timestamp(l);
+      expect(ts.i).to.equal(0xffffffff); // -1 unsigned
+      expect(ts.t).to.equal(0xfffffffe); // -2 unsigned
+    });
```

Comment:
```suggestion
context('when signed negative input is provided to the constructor', () => {
    it('t and i still return unsigned values', () => {
      const l = new BSON.Long(-1, -2);
      const ts = new BSON.Timestamp(l);
      expect(ts.i).to.equal(0xffffffff); // -1 unsigned
      expect(ts.t).to.equal(0xfffffffe); // -2 unsigned
    });
 });
```

Reply:
updated naming, and added an assumption check that the Long class does not make the value unsigned